### DISCUSSION TITLE
Response status validation per Endpoint

### DISCUSSION THREAD

**Discussion by stychu** [ecyrbe/zodios#221] [Q&A] [answered]
Is there an easy way to define valid status responses per api endpoint??
I would like to be able to use default `validateStatus` which is I believe `status >= 200 && status < 300` but when I have
`makeEndpoint` definition then I would like to explicitly allow this endpoint to NOT THROW for 200 and 400. Is it possible ?

**Comment by ecyrbe** [comment, marked as answer]
It's possible, but you need to either pass a `validateStatus` function to the [axios config](http://www.zodios.org/docs/client#request-options) or create a plugin that does it and attach the plugin to your endpoint

**Reply by stychu** [reply]
>you need to either pass a validateStatus function to the [axios config](http://www.zodios.org/docs/client#request-options)

This will behave like a global change right? I need a control over particular endpoints but keep the default global settings. Plugins seems the right choice. Thanks!
